src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -92,5 +96,25 @@ private JObject InvokeScript(JArray _params)
             CheckWitnessHashes checkWitnessHashes = _params.Count >= 2 ? new CheckWitnessHashes(((JArray)_params[1]).Select(u => UInt160.Parse(u.AsString())).ToArray()) : null;
             return GetInvokeResult(script, checkWitnessHashes);
         }
+
+        [RpcMethod]
+        private JObject GetUnclaimedGas(JArray _params)
+        {
+            string address = _params[0].AsString();
+            JObject json = new JObject();
+            UInt160 script_hash;
+            try
+            {
+                script_hash = address.ToScriptHash();
+            }
+            catch
+            {
+                script_hash = null;
+            }
```

Comment:
maybe need to check if it is invalid and return.

Reply:
I think that we should throw an exception

Reply:
Throw the exception and what.in the json return,@shargon?

Reply:
done